detox/android/detox/src/full/java/com/wix/detox/DetoxActionHandlers.kt

```diff
@@ -89,37 +89,44 @@ class CleanupActionHandler(
         } else {
             testEngineFacade.resetReactNative()
         }
-        wsClient.sendAction("cleanupDone", emptyMap<Any, Any>(), messageId)
+        outboundServerAdapter.sendMessage("cleanupDone", emptyMap(), messageId)
     }
 }
 
 class QueryStatusActionHandler(
-        private val wsClient: WebSocketClient,
+        private val outboundServerAdapter: OutboundServerAdapter,
         private val testEngineFacade: TestEngineFacade)
     : DetoxActionHandler {
 
     override fun handle(params: String, messageId: Long) {
         val data = mutableMapOf<String, Any>()
         val busyResources = testEngineFacade.getBusyIdlingResources()
-        var status = ""
 
-        if (busyResources.isEmpty()) {
-            status = "The app is idle."
-        } else {
-            status = "Busy idling resources:\n"
-            for (res in busyResources) {
-                status += "\t- ${res.name}\n"
-            }
+        data["status"] = "App synchronization debug: " +
+            if (busyResources.isEmpty()) {
+                "The app appears to be idle!"
```

Comment:
@noomorph I'm thinking about omitting the output for this use case. Makes the log dirty, and I'm not sure the reason is good enough, as the information is not very valuable. wdyt?

Reply:
Might be not the best idea, although I understand the motivation. Look, if a request does not end in 10s, should not that be suspicious? Of course, for `waitFor` case it could be nice to check whether the app was busy or not while we have been waiting. But for the other requests, the unresponsiveness can only raise suspicions.

We might although put those messages under debug loglevel on JS side, what do you think?

Reply:
Okay, left it as-is for now. Presumably, users won't see much of it, as it is mostly shown when a high-frequency debug check is employed.